### ISSUE TITLE
PHP 84 support

### ISSUE THREAD

**Issue by titken** [Microsoft/msphpsql#1526]
PHP 8.4 was just released - please compile drivers with updated module api.

**Comment by cmb69** [comment]
This is a duplicate of #1523.

**Comment by David-Engel** [comment]
MS has not abandoned PHP. We are going through some transitions internally, which has delayed things. I don't have an exact timeline for the next PHP driver release, but it's likely at least a couple quarters out.

Feel free to @ me with direct feedback. Having customer voices allows me to easily bubble those up internally to ensure appropriate prioritization of resources.

**Comment by akrabat** [comment]
@David-Engel As a data point, I have an enterprise client in the insurance space that has a key PHP application that uses SQL Server and is on Azure. They are waiting on PHP 8.4 support in the sqlsrv driver in order to complete the update. 

It would be helpful to get this released during this quarter.

**Comment by titken** [comment]
@David-Engel seems like Windows DLL drivers for PHP 8.4 was uploaded to PECL website for both [pdo_sqlsrv] and [sqlsrv] in January 2024 and you are mentioned as maintainer. 

Would you recommend using them for production?
And is there a reason they're uploaded there and not in the official driver package on Microsoft site?

**Comment by cmb69** [comment]
This ticket has been closed as duplicate of #1523; it's better to check what is going on over there (TL;DR: unfortunately no updates).

**Comment by David-Engel** [comment]
See my comment: https://github.com/microsoft/msphpsql/issues/1523#issuecomment-3140574932
